src/components/Main.jsx

```diff
@@ -15,6 +16,7 @@ const Main = () => (
       <Route path="/recent" component={RecentSnippets} />
       <Route path="/about" component={About} />
       <Route path="/sign-in" component={SignIn} />
+      <Route path="/:id" component={Snippet} />
```

Comment:
Hmm, can we somehow make it work with an optional trailing / ? E.g. now both `http://xsnippet.org/363505/` and `http://xsnippet.org/363505` links work and the latter performs a redirect to the former. My understanding is this code only works without a trailing /.

Reply:
About `Cannot GET` error seems that this is an ussue related how we start project. Currently we use `webpack-dev-server` and as far as I know this serves files from memory and not from builded files. Seems it can't work with links that not default properly, but this behaviour we can fix by extending `webpack.config.js` with next options:
```devServer: {
      historyApiFallback: true,
    },
``` Can add this in separate PR if is ok with you.

As for trailing slash I need some time to investigate this

Reply:
> Can add this in separate PR if you ok with this.

Yeah, that would be great! Thanks!

> As for trailing slash I need some time to investigate this

Cool! I do not think it's a blocker for this PR, though. Just something to keep in mind.

Reply:
@malor Please ensure you route all `/*` requests to `index.html` as otherwise we will encounter 404 in production. :)

P.S: I mean your ansible playbook for xsnippet-web :P